src/eval-frame/components/panes/declared-variables-pane.jsx

```diff
@@ -23,55 +19,21 @@ export class DeclaredVariablesPaneUnconnected extends React.Component {
   }
 
   render() {
-    const noEnvironmentVariables = !Object.keys(this.props.environmentVariables)
-      .length;
-    const noDeclaredVariables = !this.props.userDefinedVarNames.length;
-    const noVariables =
-      noEnvironmentVariables && noDeclaredVariables ? (
-        <EmptyPaneContents>No Declared Variables</EmptyPaneContents>
-      ) : (
-        undefined
-      );
-    const edvElem = !noEnvironmentVariables ? (
-      <div className="declared-variables-list">
-        <h3>Saved Environment</h3>
-        <div className="frozen-variables">
-          {Object.keys(this.props.environmentVariables).map(varName => (
-            <FrozenVariable
-              varName={varName}
-              byteLength={this.props.environmentVariables[varName][1].length}
-              key={varName}
-            />
-          ))}
-        </div>
-      </div>
-    ) : (
-      undefined
-    );
-
-    const declaredVariables = !noDeclaredVariables ? (
-      <div className="declared-variables-list">
-        <h3>User Defined Variables</h3>
-        {this.props.userDefinedVarNames.map(varName => (
-          <DeclaredVariable varName={varName} key={varName} />
-        ))}
-      </div>
-    ) : (
-      undefined
-    );
     return (
       <div className="pane-content">
-        {noVariables}
-        {edvElem}
-        {declaredVariables}
+        <div className="declared-variables-list">
```

Comment:
turns out we don't need any of the conditionals removed from above, b/c the declared variables pane will now _always_ show at least `iodide` in the user variable explorer.